apps/src/code-studio/components/progress/course_progress.jsx

```diff
@@ -24,7 +24,7 @@ const CourseProgress = React.createClass({
   propTypes: {
     professionalLearningCourse: React.PropTypes.bool,
     focusAreaPositions: React.PropTypes.arrayOf(React.PropTypes.number),
-    stages: React.PropTypes.arrayOf(stageShape),
+    stages: React.PropTypes.arrayOf(stageShape)
```

Comment:
I've actually recently developed a slight preference for trailing commas. Not sure why I chose to remove these.